src/magicbox/SuggestionsManager.ts

```diff
@@ -46,8 +83,9 @@ export class SuggestionsManager {
   ) {
     this.root = Component.resolveRoot(element);
     this.options = defaults(options, <SuggestionsManagerOptions>{
-      selectableClass: 'magic-box-suggestion',
-      selectedClass: 'magic-box-selected'
+      suggestionClass: 'magic-box-suggestion',
+      selectedClass: 'magic-box-selected',
+      previewHeaderText: 'Query result items for'
```

Comment:
This needs to be put as a string in the localization file `strings.json` and called through `l('theNameOfOfString')`

Reply:
I just noticed it already exists as `QuerySuggestPreview` in `strings.json`. Should I rename it? For now, I'll change `previewHeaderText`'s default value to be `l('QuerySuggestPreview')`.